scripts/vagrant/zeppelin-dev/README.md

```diff
@@ -0,0 +1,154 @@
+## Vagrant Virtual Machine for Apache Zeppelin
+  
+This script creates a virtual machine that launches a repeatable, known set of core dependencies required for developing Zeppelin.  It can also be used to run an existing Zeppelin build if you don't plan to build from source.  For pyspark users, this script also includes several helpful [Python Libraries and one obscure configuration to help with matplotlib plotting inside Zeppelin](#pythonextras)
+ 
+####Installing the required components to launch a virtual machine.
+
+This script requires three applications, [Ansible](http://docs.ansible.com/ansible/intro_installation.html#latest-releases-via-pip "Ansible"), [Vagrant](http://www.vagrantup.com/downloads "Vagrant") and [Virtual Box](https://www.virtualbox.org/ "Virtual Box").  All of these applications are freely available as Open Source projects and extremely easy to set up on most operating systems.
+
+### Create a Zeppelin Ready VM in 4 Steps
+
+1. Download and Install Vagrant:  [Vagrant Downloads](http://www.vagrantup.com/downloads)
+2. Install Ansible:  [Ansible Python pip install](http://docs.ansible.com/ansible/intro_installation.html#latest-releases-via-pip)  
+`sudo easy_install pip` then   
+`sudo pip install ansible`
+3. Install Virtual Box: [Virtual Box Downloads](https://www.virtualbox.org/ "Virtual Box")
+4. Type `vagrant up`  from within the `/scripts/vagrant/zeppelin-dev` directory
+
+Thats it!
+
+You can now run `vagrant ssh` and this will place you into the guest machines terminal prompt.
+
+If you don't wish to build Zeppelin from scratch, run the z-manager installer script while running in the guest VM:
+
+```
+curl -fsSL https://raw.githubusercontent.com/NFLabs/z-manager/master/zeppelin-installer.sh | bash
+```
+
+
+
+### Building Zeppelin
+
+You can now `git clone https://github.com/apache/incubator-zeppelin.git` into a directory on your host machine, or directly in your virtual machine.
+
+Cloning zeppelin into the `/scripts/vagrant/zeppelin-dev` directory from the host, will allow the directory to be shared between you host and the guest machine.
+
+Cloning the project again may seem counter intuitive, since this script likley originated from the project repository.  Consider copying just the vagrant/zeppelin-dev script from the zeppelin project as a stand alone directory, then once again clone the specific branch you wish to build.
+
+Synced folders enable Vagrant to sync a folder on the host machine to the guest machine, allowing you to continue working on your project's files on your host machine, but use the resources in the guest machine to compile or run your project.[^1]
```

Comment:
`[^1]` footnote might not be supported? http://stackoverflow.com/questions/19483975/jekyll-on-github-pages-any-way-to-add-footnotes-in-markdown